Makefile

```diff
@@ -28,28 +36,23 @@ test-kore:
 	fi
 
 test-k:
-	$(MAKE) -C src/main/k/working test-k
-
-jenkins: distclean check all test docs
+	$(MAKE) --directory src/main/k/working test-k -j$(NPROCS)
```

Comment:
Using multiple jobs here creates a race condition; right now my machine is running 16 processes of `stack build`. This could probably be mitigated by making the `test-k` target build `kore-exec` first; Stack does rebuild-avoidance but it does not do locking (intentionally, AFAICT).

Reply:
I'll make the `test-k` target depend on the `kore` target. I'm still not a fan of this stacking of phony targets, as it results in things being rebuilt too often. But that can wait for another PR.